x-pack/plugins/security_solution/public/detections/containers/detection_engine/rules/translations.ts

```diff
@@ -109,3 +109,10 @@ export const RELOAD_MISSING_PREPACKAGED_RULES_AND_TIMELINES = (
         'Install {missingRules} Elastic prebuilt {missingRules, plural, =1 {rule} other {rules}} and {missingTimelines} Elastic prebuilt {missingTimelines, plural, =1 {timeline} other {timelines}} ',
     }
   );
+
+export const RULE_EXECUTION_EVENTS_FETCH_FAILURE = i18n.translate(
+  'xpack.securitySolution.containers.detectionEngine.ruleExecutionEventsFetchFailDescription',
```

Comment:
Slightly different keys than in [my branch](https://github.com/elastic/kibana/pull/124198/files#diff-ad13f88552640f46422f0eee2f114d845f90da5837077f8fa1dd4b06320971b8R17-R23) -- I'll be sure to run `scripts/i18n_check.js --fix` once we merge so there aren't any leftover unused keys. 👍